package.json

```diff
@@ -47,8 +44,14 @@
     "webmaker-app-icons": "git://github.com/k88hudson/ionicons#teach",
     "webpack": "^1.6.0"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "mocha-phantomjs": "^3.5.3",
+    "phantomjs": "^1.9.16",
+    "simplecrawler": "^0.5.2",
+    "supertest": "^1.0.1"
+  },
   "scripts": {
+    "postinstall": "gulp postinstall",
```

Comment:
> I had to add this post-install to generate the webpack bundle (only when `NODE_ENV=production` though, otherwise it does nothing) 

Is it because running `npm start` has already included [those tasks, expect `webpack`](https://github.com/mozilla/teach.webmaker.org/pull/936/files#diff-b9e12334e9eafd8341a6107dd98510c9R242)?